StripeIdentity/StripeIdentity/Source/NativeComponents/Coordinators/VerificationSheetFlowController.swift

```diff
@@ -597,7 +623,7 @@ extension VerificationSheetFlowController: VerificationSheetFlowControllerProtoc
                 documentType: documentType,
                 sheetController: sheetController,
                 cameraSession: makeDocumentCaptureCameraSession(),
-                documentUploader: documentUploader,
+                documentUploader: documentUploader!,
```

Comment:
the force unwrap (`!`) seems like it should be safe but worth considering being cautious (force unwrap with null is a crash)

another way one could write this is:

```
        let documentUploader = DocumentUploader(
            imageUploader: IdentityImageUploader(
                configuration: .init(from: staticContent.documentCapture),
                apiClient: sheetController.apiClient,
                analyticsClient: sheetController.analyticsClient,
                idDocumentType: documentType
            )
        )
       self.documentUploader = documentUploader
```

then the local variable remains to be non-null